delta-api/pom.xml

```diff
@@ -36,6 +36,12 @@
       <artifactId>cdap-api</artifactId>
       <version>${cdap.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-api-common</artifactId>
+      <version>6.6.0-SNAPSHOT</version>
+      <scope>compile</scope>
```

Comment:
I think `compile` is default scope, you dont need to specify explicitly.

Reply:
done :)